code/game/machinery/martian_printer.dm

```diff
@@ -28,6 +34,17 @@
 
 	make_mob(O)
 
+	// Activate the cooldown
+	cooldown_active = 1
+	cooldown_time = world.time + cooldown_duration
+
+/obj/machinery/mob_printer/process()
+	..()
+	if(world.time >= cooldown_time)
```

Comment:
And if the cooldown is over, you can reset cooldown_time to 0 with initial(cooldown_time).

Reply:
initial(cooldown_time) literally does nothing when I tested it, DM even pings me with a warning saying it's a dummy line

Reply:
OK, to elaborate, I know now what you meant, but there's no reason to do that. cooldown_time is just a var that's set to the date world.time has to reach in order for the cooldown to be up. After world.time passes that, it doesn't matter if it gets changed to 0, both will have the same effect. Either way, why in the world would I do `cooldown_time = initial(cooldown_time)` when `cooldown_time = 0` does the exact same thing with less ambiguity?